### ISSUE TITLE
Trailing zero issue

### ISSUE THREAD

**Issue by DanPurdy** [sasstools/sass-lint#439]
As discovered in #438 it seems that the `no-trailing-zero` rule only works correctly when there is at least one non zero number following a decimal point.

The following is currently valid.
```scss

foo {
  margin: 1.0000000000
}
```

The following is not

```scss

foo {
  margin: 1.10000000000
}
```
